build-system/tasks/bundle-size/index.js

```diff
@@ -38,10 +36,16 @@ const {
   VERSION: internalRuntimeVersion,
 } = require('../../compile/internal-version');
 const {cyan, green, red, yellow} = require('ansi-colors');
+const {report} = require('@ampproject/filesize');
 
 const requestPost = util.promisify(require('request').post);
 
-const fileGlobs = ['dist/*.js', 'dist/v0/*-?.?.js'];
+const fileGlobs = [
```

Comment:
@danielrozenberg What do you think about grabbing this from package.json in a future change? That's where `filesize` needs it to be.

Reply:
I would insist on grabbing this list from `package.json` in this PR. It took me a while to understand from the code why you don't pass this const to `report()` at all ("where is this being used????") so the duplication between here and `package.json` is confusing...

Reply:
Makes sense, pulled this value from `package.json` via require.